packages/react/src/components/Table/StatefulTable.jsx

```diff
@@ -322,8 +322,8 @@ const StatefulTable = ({ data: initialData, expandedData, ...other }) => {
         toolbar: {
           ...view?.toolbar,
           search: {
-            ...view?.toolbar?.search,
             defaultValue: initialDefaultSearch,
+            ...view?.toolbar?.search,
```

Comment:
I think this change prevents the search term from being externally updated via the table props

Reply:
I had a worry that would happen

Reply:
ok I've undone this to address in a different fix to the onUserViewModified callback